### DISCUSSION TITLE
Removal of roads

### DISCUSSION THREAD

**Discussion by Andreweag** [kubilus1/autoortho#219] [Ideas]
The issue of XP roads not matching with photo scenery has been around for a long time, but with Ortho4XP tiles there are utilites such as 'XRoads' which remove the generation of roads in the regions where you have tiles. It could be nice to eventually have this same option for autoortho. I'd be interested to experiment with this;  does anyone know how I could manually toggle road generation in the settings.txt or something similar?

**Comment by chezhj** [comment]
I have disabled yAutoOrtho_Overlays (xOrganizer)
And I have xroads, this is how EHAM looks from a distance
![b738 - 2023-11-03 10 47 46](https://github.com/kubilus1/autoortho/assets/59411979/38f75be5-c743-41e0-ade3-8e48eab9d10b)
